src/components/Nav/NavCompoenet/Shopping/Shopping.js

```diff
@@ -0,0 +1,41 @@
+import React, { useState, useEffect } from 'react';
+import { Link } from 'react-router-dom';
+
+import ShoppingList from './ShoppingList/ShoppingList';
+import './Shopping.scss';
```

Comment:
```suggestion
import React, { useState, useEffect } from 'react';
import { Link } from 'react-router-dom';
import ShoppingList from './ShoppingList/ShoppingList';
import './Shopping.scss';
```
import 속성 순서 잘 적용해주셨습니다! 다만 줄바꿈은 없애주세요~